website/docs/secrets/manage-secrets-ui.mdx

```diff
@@ -31,13 +31,49 @@ Here, you will be prompted to enter the `External Secret Name` and the `Target K
 
 It's important to note that the chosen `SecretStore` may be a cluster-scoped `SecretStore` ie: `ClusterSecretStore` or a namespace-scoped `SecretStore`.
 
+If you choose a namespace scoped `SecretStore`, the new secret will be created on the same namespace as the `SecretStore`.
```

Comment:
what is the scenarios we have in mind when having both `secretstore` and `secret` in the same namespace? 

my basic understanding tells me that secretstore is being provisioned as part of the operator so it is a platform capability owned by platform team. however, the external secret is the manifest to allow other teams to use the capability. having both of them in the same namespace might involve that both platform teams and application team have access to the same namespace that might not be the case.

Reply:
yes you are rigth, if the team who is responsible for the secretStore creation diffrent from the teams who will add and sync thier ExternalSecrets and they don't have the access to the same namespace as platform teams so they need to use a Cluster scoped SecretStore https://external-secrets.io/v0.4.2/api-clustersecretstore/ which will be availble and can be refrenced from any namespace.